README.md

```diff
@@ -2,49 +2,58 @@
 
 [![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Elastic Stack version](https://img.shields.io/badge/ELK-6.7.0-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/376)
-[![Build Status](https://api.travis-ci.org/deviantony/docker-elk.svg?branch=x-pack)](https://travis-ci.org/deviantony/docker-elk)
+[![Build Status](https://api.travis-ci.org/deviantony/docker-elk.svg?branch=master)](https://travis-ci.org/deviantony/docker-elk)
 
-Run the latest version of the [Elastic stack](https://www.elastic.co/elk-stack) with Docker and Docker Compose.
+Run the latest version of the [Elastic stack][elk-stack] with Docker and Docker Compose.
 
-**Note**: This version has [X-Pack support](https://www.elastic.co/products/x-pack) with [X-Pack
-security](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-security.html) enabled. The [trial
-license](https://www.elastic.co/guide/en/elasticsearch/reference/current/license-settings.html) is valid for 30 days.
+It gives you the ability to analyze any data set by using the searching/aggregation capabilities of Elasticsearch and
+the visualization power of Kibana.
 
-It will give you the ability to analyze any data set by using the searching/aggregation capabilities of Elasticsearch
-and the visualization power of Kibana.
+> :information_source: The Docker images backing this stack include [Stack Features][stack-features] (formerly X-Pack)
+with [paid features][paid-features] enabled by default. The [trial license][trial-license] is valid for 30 days.
 
 Based on the official Docker images from Elastic:
 
 * [elasticsearch](https://github.com/elastic/elasticsearch-docker)
 * [logstash](https://github.com/elastic/logstash-docker)
 * [kibana](https://github.com/elastic/kibana-docker)
 
+> :information_source: Other branches in this project are available:
+>
+> * [`searchguard`](https://github.com/deviantony/docker-elk/tree/searchguard): Search Guard support
+> * [`vagrant`](https://github.com/deviantony/docker-elk/tree/vagrant): run Docker inside Vagrant
```

Comment:
We can remove the `vagrant` branch ref, I don't think it is worth maintaining anymore.

Reply:
Done